doc/classes/Label.xml

```diff
@@ -88,8 +88,8 @@
 			Controls the text's vertical alignment. Supports top, center, bottom, and fill. Set it to one of the [enum VerticalAlignment] constants.
 		</member>
 		<member name="visible_characters" type="int" setter="set_visible_characters" getter="get_visible_characters" default="-1">
-			Restricts the number of characters to display. Set to -1 to disable.
-			[b]Note:[/b] Setting this property updates [member percent_visible] based on current [method get_total_character_count].
+			The number of characters to display. If [code]-1[/code], all characters display. Useful to animate the text in a dialog box.
```

Comment:
```suggestion
			The number of characters to display. If set to [code]-1[/code], all characters are displayed. This can be useful when animating the text appearing in a dialog box.
```